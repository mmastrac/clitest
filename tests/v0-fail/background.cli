#!/usr/bin/env clitest --v0

background {
    $ exit 1
}

$ echo "hi"
! hi
