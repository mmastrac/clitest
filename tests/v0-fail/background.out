Running <root>/background.cli ...

printf "hi\n"
---
hi
---
✅ OK

(background) exit 1
---
(no output)
---
❌ FAIL: exit status: 1
 → exit 1

<root>/background.cli FAILED
Error: exit status: 1

1/1 test(s) failed
